test/whitelist/binutils/glibc.log

```diff
@@ -0,0 +1,6 @@
+#
+# XXX: Unknwon.
+#
+FAIL: indirect5c dynsym
+FAIL: indirect5d dynsym
```

Comment:
I've looked at these problems.  There is a missing check for undefweak in a couple of places, but the main problem is that we are missing a relaxation to convert pc-relative got references to local symbols to pc-relative direct references.  I stopped there, because it looks like fixing this requires a new relocation, in addition to new relaxations, and never had time to do that.  I think this is the ~jimw/ld-testsuite-pie.txt file on hifiveu017.  This is an optimization, not a correctness issue, so isn't a high priority.